Command/DropDatabaseDoctrineCommand.php

```diff
@@ -49,7 +49,12 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $connection = $this->getDoctrineConnection($input->getOption('connection'));
+        $connectionName = $input->getOption('connection');
+        if (empty($connectionName) === true) {
```

Comment:
`=== true` is useless. `empty($connectionName)` is already a boolean

Reply:
i changed it 👍